RecoMTD/DetLayers/interface/MTDDetRing.h

```diff
@@ -42,7 +42,7 @@ class MTDDetRing : public ForwardDetRingOneZ {
 private:
   typedef PeriodicBinFinderInPhi<float> BinFinderType;
   BinFinderType theBinFinder;
-  static const std::string metname;
+  static const std::string theMetname_;
```

Comment:
as I mentioned in the main thread the trailing underscore breaks consistency with naming used for existing data members.
Please drop the underscore (the other alternative is to add it to all other existing data members)